src/privileged/trackers/api.js

```diff
@@ -6,8 +6,6 @@ ChromeUtils.import("resource://gre/modules/XPCOMUtils.jsm");
 
 const { EventManager, EventEmitter } = ExtensionCommon;
 
-const {Management: {global: {tabTracker}}} = ChromeUtils.import("resource://gre/modules/Extension.jsm", {});
```

Comment:
oddly, everything under `privileged` seems to share a namespace, so I was getting errors that this was redefined. I don't like it though

Reply:
See my comment in #71, I'm hesitant to remove this entirely...